components/CancellationList.vue

```diff
@@ -76,10 +76,13 @@
         {{ this.selectedNote }}
       </v-card>
     </v-dialog>
+    <StatusModal ref="successDialog" title="Success" text="Clinic Data Reported Successfully"/>
```

Comment:
For Vue, it's typically a better practice to use `data` or `props` for reactive properties instead of `$ref`s. 

For this specific case, I would recommend passing in a `prop` from the parent. 

See[ this example](https://github.com/ourjapanlife/findadoc-frontend/blob/fad247b07039a55bf7cfd6da54b248c9f624b995/components/PendingSubmissions.vue#L5) of `deleteDialog` being used in `PendingSubmissions` parent component.